packages/voting/graphql/resolvers/Candidacy.js

```diff
@@ -18,12 +18,34 @@ module.exports = {
       return
     }
 
-    if (!candidacy.user.address.city) {
+    if (!candidacy.user.address?.city) {
       return
     }
 
     return candidacy.user.address.city
   },
+  postalCodeGeo: async (candidacy, args, { user: me, loaders }) => {
+    if (!Roles.userIsInRoles(me, ['admin', 'supporter', 'associate'])) {
+      return
+    }
+
+    if (
+      !candidacy.user.address?.postalCode &&
+      !candidacy.user.address?.country
+    ) {
+      return
+    }
```

Comment:
If either `postalCode` or `country` is available, will `postalCodeGeo` return anything? (Pondering here, if either is missing, if it should return nothing.)

Also, it should probably return `null` instead of undefined.

Reply:
(Take this, if both, `postalCode` and `country`, are required to return anything meaningful.)

```suggestion
    if (
      !candidacy.user.address ||
      !candidacy.user.address?.postalCode ||
      !candidacy.user.address?.country
    ) {
      return
    }
```